### PR TITLE
chore:  update read and help commands [FTL-8372]

### PR DESCRIPTION
**Pull request by YoussefAWasfy**

### CODE REVIEW

**Review by JonasEtzold — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -581,17 +591,37 @@ if you simply type this, the CLI will prompt you to choose from a list of availa
...
+-o, --output=(plaintext|json)  set flag to override default output format view
+```
+
+To show info of logged in user:
+
+```
+$ affinidi show user [--output json]
+```
+
 EXAMPLES
```
Comment (diegoscaro) [inline README.md:619]: let's add blank lines after each example line

File: README.md [hunk 2]
```diff
@@ -653,6 +684,34 @@ $ affinidi use --help
 $ affinidi use project --help
 ```
 
+### **affinidi analytics**
+
+The Analytics command lets you choose whether you want send anonymous usage data or not.
```
Comment (diegoscaro) [inline README.md:689]: This command lets you opt in or out of sending anonymous usage data.

File: package.json [hunk 3]
```diff
@@ -115,7 +115,23 @@
...
+        "description": "Use this command to display the details of a resource"
+      },
+      "list": {
+        "description": "Use this command to display resources that you have created or are available"
+      },
+      "create": {
+        "description": "Use this command to create a new resource"
+      },
+      "config": {
+        "description": "Use this command to configure various settings for cli"
```
Comment (diegoscaro) [inline package.json:129]: "... to configure CLI settings"

File: src/commands/analytics.ts [hunk 4]
```diff
@@ -10,7 +10,7 @@ export const OPTIN_MESSAGE = 'You have opted in to analytics'
 export const OPTOUT_MESSAGE = 'You have not opted in to analytics'
 
 export default class Analytics extends Command {
-  static description = 'Opt-in or opt-out of analytics'
+  static description = 'Use this command to opt-in or opt-out of analytics'
```
Comment (diegoscaro) [inline src/commands/analytics.ts:13]: "Use this command to opt in or out of sending anonymous usage data"

File: src/commands/config/index.ts [hunk 5]
```diff
@@ -11,15 +11,19 @@ import { CliError, getErrorOutput, Unauthorized } from '../../errors'
 export default class Config extends Command {
   static command = 'affinidi config'
 
-  static summary = 'The config commmand allows to set various settings for cli.'
+  static summary = 'Use this commmand to delete user saved configurations.'
```
Comment (diegoscaro) [inline src/commands/config/index.ts:14]: please update this following Readme

**Review by diegoscaro — APPROVED** [review]

**Pull request merged**